Makefile

```diff
@@ -10,7 +10,7 @@ PROVIDER        := pulumi-resource-${PACK}
 CODEGEN         := pulumi-gen-${PACK}
 VERSION         ?= $(shell scripts/get-version)
 PYPI_VERSION    := $(shell scripts/get-py-version)
-KUBE_VERSION    ?= v1.15.0
+KUBE_VERSION    ?= v1.16.0-rc.2
```

Comment:
Will this be bumped to a non release candidate once it's out / before this PR gets merged?

Reply:
Yep. Usually there are no changes between the final RC and the actual release, but I'll bump it when it comes out on Monday.